scripts/jwst/build_parent_sample.py

```diff
@@ -32,18 +35,15 @@ def get_pixel_scale(header):
     # Calculate the pixel scale for each direction, assuming square pixels and small angles.
     # We'll convert from degrees to arcseconds by multiplying by 3600.
     pixel_scale_x = np.sqrt(cd[0, 0] ** 2 + cd[1, 0] ** 2) * 3600  # X direction
+    pixel_scale_y = np.sqrt(cd[0, 1] ** 2 + cd[1, 1] ** 2) * 3600  # Y direction
```

Comment:
Can we not group these lines:
```suggestion
    pixel_scale = np.sqrt(cd[0] ** 2 + cd[1] ** 2) * 3600
```